server/src/api/modules/groups/group.service.js

```diff
@@ -548,6 +549,60 @@ async function findOne(groupId) {
   return group;
 }
 
+/**
+ * Update all members of a group.
+ *
+ * An update action must be supplied, to be executed for
+ * every member. This is to prevent calling jobs from
+ * within the service. I'd rather call them from the controller.
+ *
+ * Note: this is a soft update, meaning it will only create a new
+ * snapshot. It won't import from CML or determine player type.
+ */
+async function updateAllMembers(id, updateAction) {
```

Comment:
Idk where to put this review comment lol, but please don't forget to update the docs to reflect these new endpoint changes